libs/datasets/sources/can_scraper_state_providers.py

```diff
@@ -3,8 +3,28 @@
 from libs.datasets.sources import can_scraper_helpers as ccd_helpers
 
 
-def transform(dataset: ccd_helpers.CanScraperLoader):
-    variables = [
+class CANScraperStateProviders(data_source.CanScraperBase):
+    SOURCE_NAME = "CANScrapersStateProviders"
+
+    EXPECTED_FIELDS = [
```

Comment:
In the future, for Can scrapers, could we just generate this from `VARIABLES`?

Reply:
Yes, later. Maybe hard to believe :-P but I'm trying to stick to the min refactoring to get the current task done.